### DISCUSSION TITLE
没看见怎么加入啊

### DISCUSSION THREAD

**Discussion by fyniujin** [travellings-link/travellings#1557] [寻求帮助] [answered]
没看见怎么加入啊？
网站 http://www.nmwbk.com/   顶部导航已按照要求添加 开往链接 没看到怎么加入开往的方式啊，是自动获取 还是需要自己在哪里提交网址？

**Comment by linlinzzo** [comment, marked as answer]
请提交一个 issue，选择申请加入的模板。谢谢。

**Reply by fyniujin** [reply]
issue 已经提交 请审核，加入模板 没看到在哪里，劳烦指导下

**Reply by jiangyangcreate** [reply]
![image](https://github.com/travellings-link/travellings/assets/69161207/1fa91a2a-d729-411b-a6e7-1526397c5b78)
